mirascope/core/gemini/stream.py

```diff
@@ -35,7 +44,42 @@ def cost(self) -> float | None:
     def _construct_message_param(
         self, tool_calls: list[FunctionCall] | None = None, content: str | None = None
     ) -> ContentDict:
-        return {
-            "role": "model",
-            "parts": [{"type": "text", "text": content}] + (tool_calls or []),
-        }
+        """Constructs the message parameter for the assistant.
+        Gemini currently does not support tool calls for streaming.
```

Comment:
hmm, this feels bad because if they do support streaming we'll need to come back here and change this vs. what we're currently doing which is always settings tool_calls to `None` because it always will be.

This is an internal function, so why remove the functionality at all?